src/tests/suite/daos_degrade_ec.c

```diff
@@ -302,7 +302,11 @@ degrade_small_sub_setup(void **state)
 
 	rc = test_setup(state, SETUP_CONT_CONNECT, true,
 			DEGRADE_SMALL_POOL_SIZE, 6, NULL);
-	return rc;
+	if (rc)
+		print_message("It can not create the pool with 6 ranks"
+			      " probably due to not enough ranks %d\n", rc);
+
+	return 0;
```

Comment:
In this test will result as PASSED, better to use skip() so the final result will have SKIPPED test in the result instead of PASSED?
```
[==========] Running 1 test(s).
[ RUN      ] REBUILD14: rebuild large stripe object
setup: creating pool, SCM size=8 GB, NVMe size=32 GB
ERROR: dmg: pool create failed: server: code = 645 description = "pool request contains invalid ranks: 2,3"
ERROR: dmg: server: code = 645 resolution = "retry the request with a valid set of ranks"
dmg_pool_create failed, rc: -1025
It can not create the pool with 4 ranks probably due to not enough ranks -1025
state not set, likely due to group-setup issue
state not set, likely due to group-setup issue
[       OK ] REBUILD14: rebuild large stripe object
[==========] 1 test(s) run.
[  PASSED  ] 1 test(s).
```

Reply:
Hmm, this is the setup(), can we use skip() in setup() ?

Reply:
Yes it has to be in the test and not in the setup()

```
It can not create the pool with 6 ranks probably due to not enough ranks -1025
Could not run test: src/tests/suite/daos_degrade_ec.c:308: Skipped!
```